website/content/docs/secrets/gcp.mdx

```diff
@@ -211,16 +211,37 @@ below.
 For more information on creating and managing static accounts, see the
 [GCP secrets engine API docs][api] docs.
 
+## Impersonated Accounts
+
+Impersonated accounts are a way to generate a GCP access token that is granted
```

Comment:
Suggest to link to the common section that explains access tokens and be consistent with the wording (GCP -> OAuth2) used throughout the doc.

```suggestion
Impersonated accounts are a way to generate a OAuth2 [access token](/docs/secrets/gcp#access-tokens) that is granted
```